### ISSUE TITLE
flake8 and darglint-ignore-regex

### ISSUE THREAD

**Issue by cazador481** [terrencepreilly/darglint#180]
Looks like flake8 has added the cmd line option of --darglint-ignore-regex.
When putting that in the tox.ini file such as
[darglint]
darglint-ignore-regex = ^_(.*)

darlint isn't picking up the regex.
If put the `--darglint-ignore-regex='^_(.*)'` darglint is picking that up.

**Comment by terrencepreilly** [comment]
Yes, it [doesn't parse](https://github.com/terrencepreilly/darglint/blob/abc26b768cd7135d848223ba53f68323593c33d5/darglint/flake8_entry.py#L87) this value from config files.  You're welcome to submit a pull request if you think it should.
